allennlp/models/semantic_role_labeler.py

```diff
@@ -191,8 +198,43 @@ def get_viterbi_pairwise_potentials(self):
 
     @classmethod
     def from_params(cls, vocab: Vocabulary, params: Params) -> 'SemanticRoleLabeler':
-        text_field_embedder = TextFieldEmbedder.from_params(vocab, params.pop("text_field_embedder"))
-        stacked_encoder = Seq2SeqEncoder.from_params(params.pop("stacked_encoder"))
+        """
+        With an empty ``params`` argument, this will instantiate a SRL model with the same
```

Comment:
Woah. This is super weird. You would say "an SRL model", but "a Semantic Role Labeling model".  Deep.